servers/src/main/java/tachyon/worker/block/evictor/EvictorFactory.java

```diff
@@ -30,7 +32,8 @@ private EvictorFactory() {}
    * @param view BlockMetadataManagerView to pass to Evictor
    * @return the generated Evictor
    */
-  public static Evictor create(EvictorType evictorType, BlockMetadataManagerView view) {
+  public static Evictor create(EvictorType evictorType, BlockMetadataManagerView bmmView) {
```

Comment:
Could you change the argument name to `view` to be consistent with the rest of the codebase?